stack-easybib/recipes/role-phpapp.rb

```diff
@@ -1,3 +1,43 @@
+php_version = node['stack-easybib']['php_version']
+
+php_core_deps = %W(
+  php#{php_version}-apcu
+  php#{php_version}-bcmath
+  php#{php_version}-cli
+  php#{php_version}-ctype
+  php#{php_version}-curl
+  php#{php_version}-dom
+  php#{php_version}-fileinfo
+  php#{php_version}-fpm
+  php#{php_version}-gearman
+  php#{php_version}-iconv
+  php#{php_version}-intl
+  php#{php_version}-json
+  php#{php_version}-mbstring
+  php#{php_version}-memcache
+  php#{php_version}-mysqli
+  php#{php_version}-pdo-mysql
```

Comment:
Also, same here, see my previous comment on the line you removed. (Regarding `php::modules-mysqli`). 

The `modules-xx` recipes provide means to configure the extensions. They should be used where available.